lua/crates/toml.lua

```diff
@@ -774,10 +777,17 @@ function M.parse_crates(buf)
             dep_section = M.parse_section(section_text, line_nr, section_start - 1)
             dep_section_crate = nil
             if dep_section then
+                if dep_section.name then
+                    working_crate = {
```

Comment:
If there is a second dependency section with a name, completion in the first one doesn't work:

```toml
[dependencies]
axum
#   ^ typing here

[dependencies.actix-web]
version = "4.5.0"
```

It might make sense to have a list of `working_crates` instead